models/comments.model.js

```
@@ -10,3 +10,29 @@ exports.selectComments = ({ article_id }) => {
     return checkItemExists(rows) || rows;
   });
 };
+
+exports.insertComment = ({ username, body }, { article_id }) => {
+  if (username === undefined || body === undefined) {
```

PSQL should throw an error for you if the username or body is undefined.